### PR TITLE
Remove the drivers of Chinese cloud vendors

### PR DESCRIPTION
**Pull request by vickyhella**
This PR removes content about the Chinese cloud vendor drivers, including Alibaba ACK, Tencent TKE, and Huawei CCE.

Reasons to remove them:
- The `drivers.rancher.cn` website that provides these driver downloads is no longer working
- These drivers haven't been updated for at least 2 years and thus cannot work properly with Rancher Manager

JIRA ticket: SURE-5852

### CODE REVIEW

**Comment by vickyhella** [comment]
@niusmallnan please take a look and thanks!

**Review by niusmallnan — APPROVED** [review]
LGTM

**Review by kingsd041 — APPROVED** [review]
LGTM

File: docs/pages-for-subheaders/about-provisioning-drivers.md
Comment (martyav) [inline docs/pages-for-subheaders/about-provisioning-drivers.md:30]: We still have these listed on the KontainerDrivers page of the Rancher UI. I think we need to add some language advising against using them unless and until they're removed. Otherwise, we might get requests to restore the how-to guides we're removing in this PR.

Comment (martyav) [inline docs/pages-for-subheaders/about-provisioning-drivers.md:30]: We still have these listed on the KontainerDrivers page of the Rancher UI. I think we need to add some language advising against using them unless and until they're removed. Otherwise, we might get requests to restore the how-to guides we're removing in this PR.

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by btat** [comment]
From the JIRA ticket it looks like this PR has a dependency on the drivers being removed on the backend first. If available, could you add the associated rancher/rancher links to this PR. In the meantime, we may want to add `[Do not merge]` to the title or converting the PR to a draft until the backend work is done.

**Comment by niusmallnan** [comment]
I submitted some backend PRs to remove the drivers, once those PRs are merged, I'll let you know here.

**Comment by martyav** [comment]
@niusmallnan Back in February, you commented that you had submitted PRs to remove the drivers, but to hold off on merging this until that was finished. Are we clear to merge this now?

**Comment by niusmallnan** [comment]
@martyav No, those PRs have not been merged:
2.7: https://github.com/rancher/rancher/pull/40402
2.6: https://github.com/rancher/rancher/pull/40403

I am also waiting for the reply in Jira, once there is new progress, I will update here.

**Comment by btat** [comment]
Closing as the dependent issues have been closed as well.

**Pull request closed without merging**
